### PR TITLE
Add docs for table CLI and update CLI commands

### PR DESCRIPTION
**Pull request by gpang**

### CODE REVIEW

**Comment by gpang** [comment]
failed: `alluxio.server.ft.journal.MultiMasterJournalTest.testCheckpointReplay`

**Comment by gpang** [comment]
Jenkins, test this please

**Comment by gpang** [comment]
failed `alluxio.client.cli.fs.JobServiceFaultToleranceShellTest.distributedMv`

**Comment by gpang** [comment]
Jenkins, test this please

File: docs/en/basic/Command-Line-Interface.md [hunk 1]
```diff
@@ -997,3 +997,114 @@ For example, `unsetTtl` can be used if a regularly managed file requires manual
 ```console
 $ ./bin/alluxio fs unsetTtl /data/yesterday/data-not-yet-analyzed
 ```
+
+## Table Operations
```
Comment (calvinjia) [inline docs/en/basic/Command-Line-Interface.md]: Could you add a pointer in the general operations section above? (similar to how we have `fs` detailed)

Comment (gpang) [reply docs/en/basic/Command-Line-Interface.md]: Done

**Comment by gpang** [comment]
failed: `alluxio.server.ft.journal.MultiMasterJournalTest.testTtl`

**Comment by gpang** [comment]
Jenkins, test this please

**Comment by gpang** [comment]
Jenkins, test this please

**Comment by ZacBlanco** [comment]
@calvinjia @apc999 does this PR have approval for merge?

**Comment by yuzhu** [comment]
alluxio-bot, merge this please

**Comment by yuzhu** [comment]
alluxio-bot, cherry-pick this to branch-2.1 please

**Pull request merged**